### ISSUE TITLE
Visualization 

### ISSUE THREAD

**Issue by pavibalaji** [zhangqianhui/progressive_growing_of_gans_tensorflow#7]
![1200_real](https://user-images.githubusercontent.com/36034278/35934656-6f6d468a-0c3e-11e8-875f-2b24e15d5b3d.png)
![1600_real](https://user-images.githubusercontent.com/36034278/35934668-7870d3fa-0c3e-11e8-82ae-5e528d3ec503.png)
For some reason, I am able to see only a part of the faces using your code. Any idea how I can fix this? Thanks in advance.

**Comment by pavibalaji** [comment]
I see it is because of the scipy.ndimage.zoom changed it. Now it should work fine. Thanks

**Comment by zhangqianhui** [comment]
Do you use celebA HQ? If yes, you should change the code below.

```
def transform(image, npx= 64 , is_crop=False, resize_w= 64):

    # npx : # of pixels width/height of image
    if is_crop:
        cropped_image = center_crop(image , npx , resize_w = resize_w)
    else:
        cropped_image = image
        cropped_image = scipy.misc.imresize(cropped_image ,
                            [resize_w , resize_w])

    return np.array(cropped_image)/127.5 - 1
```

You need a suitable value for cropping the dataset.

**Comment by pavibalaji** [comment]
No, I am using celebA. I still couldn't figure out why it shows like this. :(

**Comment by zhangqianhui** [comment]
Have you changed the code for cropping ?

**Comment by pavibalaji** [comment]
No I haven't.

**Comment by ronslos** [comment]
I have the same problem.
The image grid that is saved to the directory PGGanCeleba are all cropped as shown above.

**Comment by zhangqianhui** [comment]
@pavibalaji  @ronslos  Can you commit your code to one repository. So, I can help you to solve this problem.

**Comment by zhangqianhui** [comment]
@ronslos  Experiments for generating 256x256 pixels images ?

**Comment by pavibalaji** [comment]
@zhangqianhui I have committed my code to my repo https://github.com/pavibalaji/Progressive_GANs_Tensorflow
Please take a look. Many thanks for your efforts :)

**Comment by zhangqianhui** [comment]
because of the scipy.ndimage.zoom ? What do you think?

**Comment by pavibalaji** [comment]
I tried without it. Unfortunately, didn't help.

**Comment by zhangqianhui** [comment]
My training is running. Now  for generating 8x8 pixels, I don't find this problem.

**Comment by pavibalaji** [comment]
you mean without scipy.ndimage.zoom?

**Comment by zhangqianhui** [comment]
No, I use this function for downsampling in my training. When does this problem happen in your trianing? Generating 8x8 pixels ? 16x16？ or others?

**Comment by pavibalaji** [comment]
Actually, in 8x8 the result isn't quite visible. Not sure if it happens exactly in 16x16 or 32x32. Is there a possibility that the cropped image is sent for training and so the GAN outputs a corner of the face? Or do you feel that, the problem is with the way the image is saved?

**Comment by zhangqianhui** [comment]
In my training just now, generative 32x32 pixels images has complete face.

**Comment by zhangqianhui** [comment]
I am running your code, Please waiting this results.

**Comment by zhangqianhui** [comment]
hello, Can you send some training photo(CelebA images) to my email? zhang163220@gmail.com

**Comment by ronslos** [comment]
Hi,
I figured out my mistake. 
I was training on the original CelebA dataset and not on the cropped version that you supply with your code.
Thanks for the help!

**Comment by zhangqianhui** [comment]
congratulate

**Comment by zhangqianhui** [comment]
@pavibalaji training on the cropped version of CelebA not the original celeba

**Comment by pavibalaji** [comment]
Thanks for pointing out. Such a relief! :)
